lib/saito/peer.js

```diff
@@ -135,409 +123,66 @@ class Peer {
 
 
     async connect(attempt = 0) {
-
-        this.socket = await this.app.networkApi.wsConnectAndInitialize(this.peer.protocol, this.peer.host, this.peer.port);
-        this.socket.peer = this;
-
-        //
-        // TODO - test -- where is publickey set
-        //
-        //console.log("start emit handshake complete");
-        //this.app.connection.emit("handshake_complete", this);
-        //console.log("done emitting handshake complete");
-
+	if (this.app.BROWSER == 1) {
```

Comment:
i think this `if else` can be simplified as below:
`this.socket = await this.app.network.initializeWebSocket(this, false, this.app.BROWSER == 1);`

Reply:
beautiful